### ISSUE TITLE
Question about copyright notice

### ISSUE THREAD

**Issue by pinkflames** [jaakkopasanen/AutoEq#790]
First of all, I'm not a lawyer, so the following are just best guesses of someone on the internet. Secondly, I know that this is an important project for the community and I want to make it clear that I'm not against projects to make the world suck less.

With that out of the way, I'm mildly perplexed at the legal status of this repository. The Github reports the project as being under the MIT license. I'm sure this means the programming language code, to which I have nothing to say. However most people do not have a deep understanding of legal issues and to them the entire project is covered by MIT license. This includes the CSV files under the many subfolders of the measurements folder.

Setting aside whether or not MIT license can be applied to them, which may or may not be an entire legal rabbit hole unto itself, the much more pressing issue is that it would seem to me that these files were taking from various publications and other sources, which is troubling for two reasons:
1. Due to the [LICENSE](https://github.com/jaakkopasanen/AutoEq/blob/master/LICENSE) file containing `Copyright (c) 2018-2022 Jaakko Pasanen` these files might be misconstrued as being covered by that copyright, when in fact they either have their original copyright or none at all (I'm not a lawyer and the copyright status of scientific data is not known to me). Either way they should probably be correctly cited according to academic standards, if nothing else.
2. Randomly examining name_index.tsv files I quickly found at least one with only dropbox URLs for source. To me this raises not so much eyebrows as the hair on the back of the neck, because that is totally not an acceptable citation or sufficient identification of the source. As such it's unclear to me if those measurement sets would even qualify as academic publications that deserve a citation rather than something which may never have been properly published in the first place and for which I can only suggest talking to a lawyer specializing in copyright or science before re-publishing. Not least because, if there's no other copyright or proper citation, it might fall under the copyright statement in the LICENSE file, which itself could be a legal problem, if real lawyers came looking.

All the best, I mainly created this issue to link it back to an [EasyEffects' issue on providing AutoEq like feature](https://github.com/wwmm/easyeffects/issues/2027) but I did not think it was appropriate to post my rambling there.

**Comment by jaakkopasanen** [comment]
You're right in that the license was intended to cover only the code, and not the data. I'll make that more clear somehow.

**Comment by photonometric** [comment]
This is an interesting question, but (and realize I'm neither a lawyer _or_ a programmer) **I'm fairly certain it's a complete non-issue.**

Yes, the MIT license applies only to the source code (and/or software documentation written by the dev, usually); this is _always_ true. Probably every project on GitHub has elements which are not copyrighted by the particular dev, nor dictated/addressed by a license governing software usage and modification. Examples of this include aesthetic presentation and generative web page elements owned by GitHub (their right to host/fork), maybe things like settings files that are "written" by either the software or the user, as well as this public interaction: _I own the copyright for this post, as well as any code I submit that gets incorporated into your project_. 

The license is a means to grant unusual rights to our work going forward, but it isn't a method to somehow invalidate, supplant, or even comprehensively catalogue the network of rightsholders involved in the source code (and discussions around it). Wider copyright laws and conventions still apply in the same way they do to everything else—except those explicitly abrogated or modified by the software license. 

Your explicit copyright claim in the source code or license is not only completely standard, but is actually (technically) redundant (though perhaps a good idea sometimes); **copyright is automatic, just like I automatically own all the posts I've made here** (pursuant to GitHub hosting terms or CC licensing, as the case may be with forum TOS). Also crucially, that _basic claim to copyright doesn't somehow imply that I authored or own quotations of users or web pages included in my posts—_nor that you are somehow claiming authorship or ownership of the vast network of open-source coding and scientific data_ (not to mention, psychoacoustics and the full history of open-source projects like Python libraries) _which your project depends upon and (legitimately) includes._ 

Even if source snippet authors expect to be credited or protected for the code they contribute, that is usually done in the source code itself, or in other project histories—just as the sources of the test data are credited here alongside their data. Not to put too fine a point on it, but simply claiming copyright for content you absolutely _already own the copyright for_, does not obligate you to list 1000 other copyrights in the same breath/license (especially since copyright doesn't apply—see below).

**Regarding the "test data," my understanding is that it's not copyrightable**. It's the same as writing a cookbook: one copyrights the specific wording, aesthetic presentation, and charming anecdotes about mother's cooking and being a sous-chef. But it's not possible to "copyright" a list of food ratios (or general descriptions/instructions)—and this appears to apply to raw test/scientific data, as well. Since the data used here is all just raw numerical measurements (I think), digested into CSV or other formats, I don't think there can conceivably be any problem. If you did a 1:1 copy of the data tables and other presentation elements from wherever these tests were published, that might be a problem (just like lifting a whole chapter of a cookbook). 

Legally speaking, I don't think you're under any obligation or expectation of exhaustively crediting sources for every contributed element (especially raw data describing a test process of a physical object), just because you claimed copyright over the content you created. Though granted, it's a nice thing to do, and also essential for this kind of evidence-based database. 

If I had to make a suggestion in this area, I'd just point out that a lot of new users do come to this github from other sites or the AutoEq "landing page" domain, and this whole project and even the basic EQ/software audio processes can be hugely bewildering. As you've said, this repo is primarily aimed at devs and/or serious enthusiasts/contributors. It might be well to include a few sentences of explanation about what the software does, why we like it, and (as suggested here) emphasize the big contributions of those audio measurement addicts. Yes, this is mentioned on the repo readme, but kind of in a way that only makes sense if you're already gone through the whole process and digested introductory info that lives elsewhere.

**Comment by pinkflames** [comment]
That was exactly my point but from the opposite perspective. In many legal frameworks copyright is automatic, which means you __must__ specify that you're quoting or including someone else's work, else it automatically falls under your copyright and you just committed plagiarism.

Additionally, I think, you're mixing up patents with copyright. This is far outside of my expertise but you probably can't patent fragrances, fashion, food or even color. Yet it's possible to have copyright over the cookbook or a color system such as the Pantone catalogues. My concern is purely about copyright and attribution rather than patents, since plenty of people seem to look at the license for the repository and assume it's fair game to include this into compiled software, when in reality it would require that software to then also cite the sources at the very least and even then I'd probably be weary of directly including such dataset myself but I'm paranoid like that.

Regarding scientific measurements, as I said, to a non-lawyer like me it seems like a grey area but it might actually be quite clear to a real lawyer specializing in copyright. For example, it's common knowledge that water has specific heat capacity of 4200 J/kg*K and I do not need to cite my middle school physics textbook or Bing where I double checked it, however I don't think you can just go around lifting tables with such data without attributing them to the source. And I imagine copying such tables in part or full from modern textbooks would result in DMCA or similar action taken against you, especially if they had a way of proving that you took that data from their work.

No one owns the facts but, I believe, they can own their work such as putting together tables or other data e.g. maps of the physical world, which are famously copyrightable via errors, be them intentional or not. Likewise going back to my example of water's specific heat capacity, even if the rough value of 42E2 is common knowledge, surely a very accurate new measurement would be copyrightable in no small part due to how much effort and resources it would likely require. But of course it's not an issue in academia, since no one would claim a breakthrough measurement by another team as their own and would definitely cite them as the source.

As a software engineer by training who briefly worked in research rather than a lawyer, I do not know where the line between fair use and plagiarism is but I'm quite certain that making sure things are properly attributed to an academic standard is a good start and maybe enough.
